src/contributor.js

```diff
@@ -1,13 +1,56 @@
+import Francisco from "./assets/images/Francisco.png"
+import Elijah from "./assets/images/Elijah.png"
+import Jason from "./assets/images/Jason.png"
+import Gene from "./assets/images/Gene.png"
+import Nicole from  "./assets/images/Nicole.png"
+import Will from "./assets/images/Will.png"
+import Luis from "./assets/images/Luis.jpg"
+import Ahmed from "./assets/images/Ahmed.png"
+import Don from "./assets/images/Don.png"
+import Logan from "./assets/images/Logan.png"
+import Sarah from "./assets/images/Sarah.png"
+import Nate from "./assets/images/Nate.png"
+import Kyle from "./assets/images/Kyle.png"
+import Christian from "./assets/images/Christian.png"
+
```

Comment:
Even though it seems odd in English, the imports should be in lowercase. And uppercase variable implies a class. I would make the file names lowercase too.